### PR TITLE
idxd-config: use distroless:base as base

### PR DESCRIPTION
**Pull request by tkatila**

### CODE REVIEW

File: build/docker/templates/intel-idxd-config-initcontainer.Dockerfile.in [hunk 1]
```diff
@@ -1,13 +1,26 @@
-FROM debian:unstable-slim
+FROM debian:unstable-slim as build
```
Comment (mythi) [inline build/docker/templates/intel-idxd-config-initcontainer.Dockerfile.in:1]: we will want to keep this aligned with what the distroless is about
```suggestion
FROM debian:stable-slim as build
```

File: build/docker/intel-idxd-config-initcontainer.Dockerfile [hunk 2]
```diff
@@ -14,12 +14,22 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 ###
-FROM debian:unstable-slim
-COPY ./LICENSE /licenses/intel-device-plugins-for-kubernetes/LICENSE
+FROM debian:stable-slim as build
 RUN apt-get update && apt-get install -y --no-install-recommends accel-config jq && rm -rf /var/lib/apt/lists/\*
+RUN mkdir -p /idxd-init/scratch
+ARG LIBS="libjson libaccel libjq libtinfo libonig libacl libselinux libpcre2-8"
+ARG EXECS="bash env accel-config jq sed grep"
```
Comment (mythi) [inline build/docker/intel-idxd-config-initcontainer.Dockerfile:21]: maybe worth thinking if it made sense to bring in toybox here too?

Comment (tkatila) [reply build/docker/intel-idxd-config-initcontainer.Dockerfile:21]: Replacing the debian tools with toybox or just having toybox as it's convenient?

Comment (mythi) [reply build/docker/intel-idxd-config-initcontainer.Dockerfile:21]: replacing bash, env, sed and grep from Debian with toybox

File: build/docker/intel-idxd-config-initcontainer.Dockerfile [hunk 3]
```diff
@@ -14,12 +14,22 @@
...
+RUN mkdir /tmp/libs && for l in ${LIBS}; do cp "/lib/x86_64-linux-gnu/${l}"* /tmp/libs/; done
+RUN mkdir /tmp/bins && for b in ls ${EXECS}; do cp "/usr/bin/${b}"* /tmp/bins/; done
```
Comment (tkatila) [inline build/docker/intel-idxd-config-initcontainer.Dockerfile:23]:
```suggestion
RUN mkdir /tmp/bins && for b in ${EXECS}; do cp "/usr/bin/${b}"* /tmp/bins/; done
```
Remove `ls` as it is not needed.

**Comment by tkatila** [comment]
At this point I don't see way forward with this PR. I'll close it.

**Pull request closed without merging**